modules/ibi-installer-configuration-config.adoc

```diff
@@ -45,4 +45,12 @@ The name of the interface must match the actual NIC name as shown in the operati
 
 |`releaseRegistry`|`string`| Specifies the container image registry that you used for the release image of the seed cluster. 
 
+|`nodeLabels` |`string`| Specifies the node labels for the {sno} node, for example:
+[source,yaml]
+----
+nodeLabels:
+  node-role.kubernetes.io/edge: "true"
+  environment: production
```

Comment:
Maybe lets have same syntax for value?
like if "true" than "production" ?

Reply:
Updated to `production: true` if this work for you